sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -47,6 +52,17 @@ export interface RetryOptions {
   // isExponential?: boolean;
 }
 
+export function getRetryAttemptTimeoutInMs(retryOptions: RetryOptions | undefined): number {
```

Comment:
This is a utility function. Can we move this to a utility file under the `util` folder?

Reply:
If we plan to have utility/helper functions like this for figuring out defaults for retry related properties, then there is merit to do the same for `retryInterval` as well.